test/extended/router/gatewayapicontroller.go

```diff
@@ -253,6 +251,9 @@ var _ = g.Describe("[sig-network-edge][OCPFeatureGate:GatewayAPIController][Feat
 
 		g.By("Checking the http route using the default gateway is accepted")
 		assertHttpRouteSuccessful(oc, gw, "test-httproute")
+
+		g.By("Validating the http connectivity to the backend application")
+		assertHttpRouteConnection(oc, defaultRoutename, gw)
```

Comment:
`defaultRoutename` seems to be misnamed. The route name here is `test-httproute`. What `defaultRoutename` carries is the route's hostname. Can you please correct this?

Reply:
done